### PR TITLE
Adds hashtags merged with develop

### PR DESCRIPTION
**Pull request by science-girl**

### CODE REVIEW

File: dna/clutter/clutter.js [hunk 1]
```diff
@@ -329,6 +343,18 @@ function getPostsBy(handles) {
   return posts;
 }
 
+function getPostsWithHashtag(input) {
+  var hashtag = input[0];
```
Comment (philipbeadle) [inline dna/clutter/clutter.js]: Any reason to use an array here?

File: test/hashtags.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (philipbeadle) [inline test/hashtags.json]: Love the tests. Only suggestion is lets use much more descriptive Conveys so we can use these as API docs.

File: dna/clutter/clutter.js (hunk 1 above)
Comment (science-girl) [reply dna/clutter/clutter.js]: Nope, removed

File: test/hashtags.json (hunk 2 above)
Comment (science-girl) [reply test/hashtags.json]: Tried to make them a little more descriptive - feel free to suggest specific ones that might need more work :-)

**Pull request merged**